src/page/GroupCreate.tsx

```diff
@@ -61,20 +66,53 @@ const GroupCreate = () => {
   };
 
   // 모든 데이터가 채워지면 그룹버튼 생성 활성화
-  useEffect(() => {
-    const allFieldsFilled = (
-      Object.keys(inputs) as Array<keyof typeof inputs>
-    ).every((key) => key === 'secretKey' || inputs[key] !== '');
-    setIsActive(allFieldsFilled || (isPublicGroup && allFieldsFilled));
-  }, [inputs, isPublicGroup]);
-
-  const handleGroupCreate = () => {
-    navigate('/group-complete', {
-      state: {
-        groupPassword: inputs.secretKey,
-        thumbnailUrl: previewImage as string,
-      },
-    });
+  // useEffect(() => {
+  //   const allFieldsFilled = (
+  //     Object.keys(inputs) as Array<keyof typeof inputs>
+  //   ).every((key) => key === 'secretKey' || inputs[key] !== '');
+  //   setIsActive(allFieldsFilled || (isPublicGroup && allFieldsFilled));
+  // }, [inputs, isPublicGroup]);
+
+  const isActive =
+    title !== '' &&
+    num !== '' &&
+    (isPublicGroup || secretKey !== '') &&
+    intro !== '' &&
+    group !== '';
```

Comment:
요게 결국 아래와 같은 형태 아닐까요!!
```suggestion
  const isActive =
    title &&
    num &&
    (isPublicGroup || secretKey) &&
    intro &&
    group;
```

Reply:
맞네요..! 감사합니다 !

Reply:
알려준 코드대로 했을 때 타입 에러가 발생해서 제가 바꾼걸 설명드리지 못했습니다 ! CreateButton 중에 isActive 라는 props를 받는데 이 props는 boolean 을 받는 props 입니다. 알려주신 코드도 분명 불리언이긴 한데, 차이점을 찾아보니 저희가 지금 입력하는 칸들이 다 "" 문자열 이다보니 하나라도 falsy 값이 나오면 "" false로 나오는게 아니라 빈 문자열인 "" 으로 인식해서 타입 에러가 뜹니다 ! 

isActive 라는 타입 자체를 string으로 바꾸면 문제가 되진 않겠지만, CreateButton 자체가 boolean 인지 아닌지 판별하면서 버튼 색상이 바뀌는 타입이라 바꾸는게 맞는지 싶어서 고민이네요 ㅠㅠ 

<img width="582" alt="image" src="https://github.com/user-attachments/assets/28532c96-7ae4-4659-8c58-42e57534ec53">